### PR TITLE
Hotfix/1.2.4

### PR DESCRIPTION
**Pull request by denis-yuen**
Do not close, this is for code review and CI

### CODE REVIEW

File: dockstore-launcher/src/main/java/io/dockstore/common/FileProvisioning.java [hunk 1]
```diff
@@ -296,27 +322,58 @@ public void registerOutputFile(String srcPath, FileInfo info) {
      *
      * @param srcPath            source file
      * @param destPath           destination file
-     * @param metadata           metddata associated with one file
+     * @param metadata           metaddata associated with one file
```
Comment (garyluu) [inline dockstore-launcher/src/main/java/io/dockstore/common/FileProvisioning.java]: typo?

File: dockstore-launcher/src/main/java/io/github/collaboratory/LauncherCWL.java [hunk 2]
```diff
@@ -251,27 +252,37 @@ private void prepUploadsHelper(Map<String, Object> inputsOutputs, final Map<Stri
...
+                } else {
+                    assert (jsonParameters instanceof List);
+                    for (Object entry : (List)jsonParameters) {
+                        if (entry instanceof Map) {
+                            handleOutputFile(fileMap, cwlID, (Map<String, Object>)entry);
                         }
                     }
                 }
+            } else {
+                System.out.println("WARNING: Output malformed for \"" + cwlID + "\" provisioning by default to working directory");
```
Comment (garyluu) [inline dockstore-launcher/src/main/java/io/github/collaboratory/LauncherCWL.java]: We have the Argument Utility, might as well use it when possible.

Comment (denis-yuen) [inline dockstore-launcher/src/main/java/io/github/collaboratory/LauncherCWL.java]: Fair enough

Comment (denis-yuen) [inline dockstore-launcher/src/main/java/io/github/collaboratory/LauncherCWL.java]: One wrinkle, the ArgumentUtility is in the client module which depends on launcher and not vice versa. 
This is probably best done as a followup in the refactored 1.3 alpha https://github.com/ga4gh/dockstore/issues/796

**Pull request merged**